spacy/lang/char_classes.py

```diff
@@ -17,6 +19,8 @@
 
 _telugu = r"\u0C00-\u0C7F"
 
+_tigrinya = r"\u1200-\u137F"
```

Comment:
I suggest to call this block of Unicode characters `_ethiopic` (https://en.wikipedia.org/wiki/Ethiopic_(Unicode_block)) instead of  `_amharic` / `_tigrinya`, also to avoid having the same block twice.

Reply:
on this one, I would highly recommend leaving it as is, as there are political implications. 😞

Reply:
I can understand that, but "Ethiopic" is the official name of this Unicode block, right?

Reply:
I agree. changed it to _ethiopic